extensions/skins/conversation_v1/Conversation.js

```diff
@@ -410,10 +410,14 @@ oppia.directive('conversationSkin', [function() {
 
           $scope.explorationCompleted = oppiaPlayerService.isStateTerminal(
             stateName);
+
+          $scope.answerClassificationData =
```

Comment:
I did this to pass answerClassificationData into the interaction. How should I do it?